data/transport missions.txt

```diff
@@ -1975,3 +1975,40 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "The Broker"
+	description "Transport the broker's enforcers to <destination> by <day>."
+	blocked "You'll need more room to house the hired thugs. Return when you have <capacity>."
+	deadline
+	cargo "Living necessities" 5
+	passengers 15
+	minor
+	to offer
+		not "year" == 3013
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			'As you're walking through the spaceport searching for a bar or pub, someone catches your eye. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You stop for a moment and look closer; he seems familiar in some way. Eventually, his gaze settles on you, making eye contact. He blinks a few times in surprise, then begins moving through the crowd towards you.'
+			choice
+				'	(Turn the other way and lose the man in the crowd.)'
+					decline
+				'	(Wait for him to reach you.)'
+				'	(Walk forwards to meet him.)'
+			'	When you and the man come face to face, he studies you for a moment. You realize who he is when he begins speaking in a smooth, mechanical voice: "Yes, I thought it was you. It's Captain <last>, correct? I remember you being quite excited to purchase your first ship, even with the somewhat outageous interest rate I gave you." It's the loan broker. The broker who gave you the money to purchase your first ship, and the one who so eagerly snatched up every payment you had given to him. "No matter," he continues. "It seems you've become quite successful."'
+			'	Seeing your unimpressed - and somewhat uninterested - look, he decides to get to the point. "I've found myself in a surprising predicament, and I would pay well to be rid of it. You see, one of my... "clients" hasn't been very proactive in paying their dues on time. This wouldn't be a problem for any normal client, but this man happens to be in charge of a small street gang on <destination>. Of course, there's a simple fix: send a group of paid enforcers to convince the client to be more... accountable, let's say."'
```

Comment:
Same issue here on drama with the fake em-dashes in the first sentence.
`Seeing your unimpressed - and somewhat uninterested - look`
The player doesn't necessarily feel this way, we're not trying to impose an opinion or personality on him.

`he decides to get to the point.` I think this is too omniscient for Endless Sky's narrative.

`one of my... "clients"`
Single quotes when you have a quote inside a quote.